t/lib/server.lua

```diff
@@ -46,6 +46,14 @@ local function inject_headers()
     end
 end
 
+local function str_split(str, reps)
```

Comment:
Let's add the mock API in the test file like:
https://github.com/apache/apisix/blob/4feab8e6bf1cf8ffd16440848b693ce14a792fbc/t/plugin/clickhouse-logger.t#L39

It's not commonly used.

Reply:
done